Dockerfile

```diff
@@ -1,9 +1,7 @@
-FROM python:3.5
+FROM python:3.6
```

Comment:
The framework should support python 3.5, is there something that is not supported?

Reply:
The Dockerfile i submitted was based on 3.6, wasn't able to get to run with 3.5 I think there's a problem f strings

Reply:
I have updated it to 3.5 and removed all f-strings to fix these issues.

Reply:
I will merge this if you can re-submit w/ python:3.5

Reply:
some type annotations doesn't work on python 3.5 if you run the examples notebooks

Reply:
Working on a fix for the type annotations, for now let's switch this to 3.5.